ophyd/ophydobj.py

```diff
@@ -38,9 +38,11 @@ class StatusBase:
     def __init__(self):
         super().__init__()
         self._lock = RLock()
-        self._cb = None
+        self._callbacks = set()
```

Comment:
:imp:  Using a set also prevents having the same callback called twice.  Not clear if this is a feature or a bug.